com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -710,7 +710,12 @@ public List<TaskResult> build(IProgress monitor, String... commands) throws IOEx
             }
             return doBuild(monitor, commands);
         } catch (CompileExceptionError e) {
+
             String s = Bob.logExceptionToString(MultipleCompileException.Info.SEVERITY_ERROR, e.getResource(), e.getLineNumber(), e.toString());
+            if (s.contains("NullPointerException")) {
+                e.printStackTrace(System.err); // E.g. when we happen to do something bad when handling exceptions
+            }
```

Comment:
I think if we're about to exit, and has hidden the true error, we'd rather like to see it.
I'm not saying this is the best approach, merely that it's better that before (where we had to guess)

Reply:
It is not beautiful, but I agree, it is better than before.